dan_layer/engine/tests/test.rs

```diff
@@ -293,6 +293,16 @@ fn test_random() {
     assert_ne!(value, vec![0; 300]);
 }
 
+#[test]
+#[should_panic(
+    expected = "called `Result::unwrap()` on an `Err` value: Transaction failed: Execution failure: RuntimeError: \
+                unreachable\n    at tari_free (<module>[327]:0x2390c)"
```

Comment:
This message is very specific, if we change the template it will differ. I test this on ubuntu and windows to make sure it passes on both.

Reply:
You could also use `test.execute_expect_failure` and inspect the error